addon/components/paper-switch.js

```diff
@@ -1,3 +1,5 @@
+/* globals Hammer */
```

Comment:
Oh okay, I didn't know that.  I'll update the places I've used Hammer to import it the right way.